src/navigation/OcTreeBase.js

```diff
@@ -407,13 +434,13 @@ ROS3D.OcTreeBase.prototype._buildFaces = function () {
     _insertFace: function (face, pos, size, color) {
       const indexCount = this.vertices.length / 3;
 
-      for (let vertex of face.vertices) {
+      face.vertices.forEach(function(vertex) {
```

Comment:
Why change all the for-of loops to forEach?

Reply:
I think it was producing linting warnings/errors.

Reply:
Fixed in the other PR